sparse_strips/vello_hybrid/examples/webgl/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+            format: surface_format,
+            width,
+            height,
+            present_mode: wgpu::PresentMode::Fifo,
+            alpha_mode: wgpu::CompositeAlphaMode::Opaque,
+            desired_maximum_frame_latency: 2,
+            view_formats: vec![],
+        };
+        surface.configure(&device, &surface_config);
+
+        let renderer = vello_hybrid::Renderer::new(
+            &device,
+            &vello_hybrid::RendererOptions {
+                format: surface_format,
+            },
+        );
+
+        Self {
+            renderer,
+            device,
+            queue,
+            surface,
+        }
+    }
+}
+
+/// Creates a `HTMLCanvasElement` of the given dimensions and renders the given `Scene` into it.
+#[cfg(target_arch = "wasm32")]
+pub async fn render_scene(scene: vello_hybrid::Scene, width: u16, height: u16) {
+    console_error_panic_hook::set_once();
+    console_log::init_with_level(log::Level::Debug).unwrap();
+    let canvas = web_sys::Window::document(&web_sys::window().unwrap())
+        .unwrap()
+        .create_element("canvas")
+        .unwrap()
+        .dyn_into::<web_sys::HtmlCanvasElement>()
+        .unwrap();
+    canvas.set_width(width as u32);
+    canvas.set_height(height as u32);
+    // Add canvas to body
+    web_sys::Window::document(&web_sys::window().unwrap())
+        .unwrap()
+        .body()
+        .unwrap()
+        .append_child(&canvas)
+        .unwrap();
+
+    let RendererWrapper {
+        mut renderer,
+        device,
+        queue,
+        surface,
+    } = RendererWrapper::new(canvas).await;
+
+    let params = vello_hybrid::RenderParams {
+        width: width as u32,
+        height: height as u32,
+    };
+    renderer.prepare(&device, &queue, &scene, &params);
+
+    let surface_texture = surface.get_current_texture().unwrap();
+    let surface_texture_view = surface_texture
+        .texture
+        .create_view(&wgpu::TextureViewDescriptor::default());
+
+    let mut encoder =
+        device.create_command_encoder(&wgpu::CommandEncoderDescriptor { label: None });
+    {
+        let mut pass = encoder.begin_render_pass(&wgpu::RenderPassDescriptor {
+            label: None,
+            color_attachments: &[Some(wgpu::RenderPassColorAttachment {
+                view: &surface_texture_view,
+                resolve_target: None,
+                ops: wgpu::Operations {
+                    load: wgpu::LoadOp::Clear(wgpu::Color::WHITE),
+                    store: wgpu::StoreOp::Store,
+                },
+            })],
+            depth_stencil_attachment: None,
+            occlusion_query_set: None,
+            timestamp_writes: None,
+        });
+        renderer.render(&scene, &mut pass, &params);
+    }
+
+    queue.submit([encoder.finish()]);
+    surface_texture.present();
+}
+
+/// Draws a blue triangle into the given `Scene`.
+pub fn draw_triangle(ctx: &mut vello_hybrid::Scene) {
```

Comment:
It might be nice to embed the GhostScript tiger SVG into the WASM binary, and display that. I don't think it would require significantly more code?

Reply:
Added ghosttiger in https://github.com/linebender/vello/pull/872/commits/03c2ef38ae4b82a3df9177b70b471beae861a0a2

![image](https://github.com/user-attachments/assets/c93b55b2-b530-4959-b819-95ed51ecbbe3)